Create varied-vancouver (2).geojson

rayraymcc opened the pull request:
mount seymour

geoglrb commented:
Hi! So what you'll need to do here is to make sure that you overwrite the old `varied-vancouver.geojson` file with your new data before issuing a pull request. This can be accomplished as literally as deleting the varied-vancouver.geojson file and renaming the `varied-vancouver (2).geojson` file to be `varied-vancouver.geojson` before committing and issuing the pull request.  At the moment, because the file you are trying to contribute back is named something else, Github isn't wanting to modify our group file, it's wanting to just make a new file with your point alongside our data, not contribute your data to ours. Apologies, but can you issue another pull request? It'll be great to get your data. Thanks!

rayraymcc commented:
Hi again! No worries, I want to get this right. I tried again and am
crossing my fingers that it worked. I deleted and renamed so many files
that if it doesn't work again I feel like I should start over again using
the information you have given me. Thank you so much for your time and
patience.

On Wed, Jan 20, 2021 at 10:28 AM Luke Bergmann <notifications@github.com>
wrote:

> Hi! So what you'll need to do here is to make sure that you overwrite the
> old varied-vancouver.geojson file with your new data before issuing a
> pull request. This can be accomplished as literally as deleting the
> varied-vancouver.geojson file and renaming the varied-vancouver
> (2).geojson file to be varied-vancouver.geojson before committing and
> issuing the pull request. At the moment, because the file you are trying to
> contribute back is named something else, Github isn't wanting to modify our
> group file, it's wanting to just make a new file with your point alongside
> our data, not contribute your data to ours. Apologies, but can you issue
> another pull request? It'll be great to get your data. Thanks!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/UBC-GEOB472-Spring2021/varied-vancouver/pull/24#issuecomment-763844569>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASQOJZO6SAGFT63P43T6KATS24OENANCNFSM4WLBHBIA>
> .
>
